server/server.ts

```diff
@@ -1,21 +1,23 @@
-require('dotenv').config();
 import fs from 'fs';
-import util from 'util';
-import express from 'express';
-import Moniker from 'moniker';
-import Youtube from 'youtube-api';
+import http from 'http';
+import https from 'https';
 import os from 'os';
+import util from 'util';
+
 import cors from 'cors';
+import express from 'express';
 import Redis from 'ioredis';
-import https from 'https';
-import http from 'http';
+import Moniker from 'moniker';
 import socketIO from 'socket.io';
-import { searchYoutube } from './utils/youtube';
+
 import { Room } from './room';
 import { getRedisCountDay } from './utils/redis';
-import { Scaleway } from './vm/scaleway';
-import { Hetzner } from './vm/hetzner';
+import { searchYoutube } from './utils/youtube';
 import { DigitalOcean } from './vm/digitalocean';
+import { Hetzner } from './vm/hetzner';
+import { Scaleway } from './vm/scaleway';
+
+require('dotenv').config();
```

Comment:
I think this only has to be done once at the top of this file (and it's probably best to move it before the imports)